package.json

```
@@ -8,14 +8,17 @@
     "prepare": "husky",
     "generate:typechain": "typechain --target ethers-v6 --out-dir generated/typechain './node_modules/@iexec/poco/build/contracts/*.json' --ts-nocheck",
     "build": "rm -rf generated && rm -rf build && npm run generate:typechain && graph codegen && graph build --network ${NETWORK_NAME:-bellecour}",
-    "test": "graph test",
+    "test": "npm run test:unit && npm run test:e2e",
+    "test:unit": "graph codegen && graph test",
```

```suggestion
    "test:unit": "graph test",
```
`codegen` already runs in `build` task.